test/Microsoft.ML.SEAL/SEALTests.cs

```diff
@@ -0,0 +1,72 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.RunTests;
+using Microsoft.Research.SEALNet;
+using Xunit;
+
+namespace Microsoft.ML.Tests.SEAL
+{
+    public sealed class SEALTests : TestDataPipeBase
```

Comment:
> public sealed class SEALTests : TestDataPipeBase [](start = 3, length = 49)

Add a test that mimics real world scenario:

Pipeline A takes plain data and encrypts it.
Pipeline B accepts encrypted data and performs inference on it.
Pipeline C accepts encrypted inferred data and decrypts it. 

In the samples I hope the above is done using multiple processes and IPC.